opentok/opentok.py

```diff
@@ -338,18 +338,30 @@ def start_archive(self, session_id, has_audio=True, has_video=True, name=None, o
         :param OutputModes output_mode: Whether all streams in the archive are recorded
           to a single file (OutputModes.composed, the default) or to individual files
           (OutputModes.individual).
+        :param String resolution (Optional): The resolution of the archive, either "640x480" (SD, the default)
+          or "1280x720" (HD). This parameter only applies to composed archives. If you set this
+          parameter and set the output_mode parameter to OutputModes.individual, the call to the
+          start_archive() method results in an error.
 
         :rtype: The Archive object, which includes properties defining the archive,
           including the archive ID.
         """
         if not isinstance(output_mode, OutputModes):
             raise OpenTokException(u('Cannot start archive, {0} is not a valid output mode').format(output_mode))
 
+
+        if resolution:
+            if output_mode == OutputModes.individual:
```

Comment:
all this logic is handled by the api server, that will return a 400 with a descriptive message... Would you like to rely on that (and make the SDK simpler)?

Reply:
I'm happy to take these checks out if that's what we want 👍.
That would make it more flexible to future changes in the API (new resolution values or if we allow resolution for individual mode) so sounds good to me.
@msach22 wdyt?

Reply:
@maikthomas @jpujol I think it's better to have the SDK raise an exception if a resolution is supplied for the individual mode, but I agree that we should not check the resolution values and let the API server handle that because those values are subject to change.